WordPress/src/main/res/layout/login_email_password_screen.xml

```diff
@@ -1,126 +1,72 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
-
-    <include
-        android:id="@+id/toolbar"
-        layout="@layout/toolbar_login" />
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingLeft="@dimen/margin_extra_large"
+    android:paddingStart="@dimen/margin_extra_large"
+    android:paddingRight="@dimen/margin_extra_large"
+    android:paddingEnd="@dimen/margin_extra_large"
+    android:layout_marginBottom="@dimen/margin_extra_large">
+
+    <TextView
+        style="@style/LoginTheme.TextLabel"
+        android:id="@+id/label"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_extra_large"
+        android:layout_marginBottom="@dimen/margin_extra_large"
+        tools:text="@string/enter_site_address" />
 
-    <ScrollView
+    <LinearLayout
```

Comment:
The nested ViewGroup (LinearLayout in this case) is needed for easy vertical centering. Notice that in your example above, the parent `android:gravity="center_vertical"` doesn't actually work since in a RelativeLayout, the children are the ones that need to define it.

That said, the LinearLayouts could be replaced with RelativeLayouts but, in one hand the optimisation is minimal (simple screen without a list view embedding the LinearLayouts) and on the other hand LinearLayouts are simple to code (no noisy pairs of `toRightOf`+`toEndOf`). Let me know if you still want me to do that change though.